AnswerKingAPI.yml

```diff
@@ -505,6 +583,18 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ProblemDetails'
+        '404':
+          description: Category with the given id does not exist
```

Comment:
Should this be `Category or Tag with the given id does not exist`?

Reply:
changed